common/persistence/sql/sqlplugin/sqlite/plugin.go

```diff
@@ -22,6 +22,67 @@
 
 package sqlite
 
+import (
+	"github.com/iancoleman/strcase"
+	"github.com/jmoiron/sqlx"
+
+	"github.com/uber/cadence/common/config"
+	"github.com/uber/cadence/common/persistence/sql/sqldriver"
+	"github.com/uber/cadence/common/persistence/sql/sqlplugin"
+)
+
 const (
 	PluginName = "sqlite"
 )
+
+// SQLite plugin provides an sql persistence storage implementation for sqlite database
+// Mostly the implementation reuses the mysql implementation
+// The plugin supports only in-memory sqlite database for now
+type plugin struct{}
+
+var _ sqlplugin.Plugin = (*plugin)(nil)
+
+// CreateDB wraps createDB to return an instance of sqlplugin.DB
+func (p *plugin) CreateDB(cfg *config.SQL) (sqlplugin.DB, error) {
+	return p.createDB(cfg)
+}
+
+// CreateAdminDB wraps createDB to return an instance of sqlplugin.AdminDB
+func (p *plugin) CreateAdminDB(cfg *config.SQL) (sqlplugin.AdminDB, error) {
+	return p.createDB(cfg)
+}
+
+// createDB create a new instance of DB
+func (p *plugin) createDB(cfg *config.SQL) (*DB, error) {
+	conns, err := sqldriver.CreateDBConnections(cfg, func(cfg *config.SQL) (*sqlx.DB, error) {
+		return p.createSingleDBConn(cfg)
+	})
```

Comment:
```suggestion
	conns, err := sqldriver.CreateDBConnections(cfg, p.createSingleDBConn)
```
Please verify, but I think this should be the same

Reply:
thanks, applied 👍